contracts/token/LimitedTransferMiniMeToken.sol

```diff
@@ -0,0 +1,68 @@
+pragma solidity ^0.4.13;
+
+
+import "./MiniMeToken.sol";
+
+
+/**
+ * Limited MiniMe transfer Token as deployed by https://fundrequest.io
+ * Davy Van Roy
+ * Quinten De Swaef
+ */
+contract LimitedTransferMiniMeToken is MiniMeToken {
+
+  bool public limitedTransfersEnabled;
+
+  address public contractAddress;
+
+  mapping (address => bool) public limitedTransferAddresses;
+
+  function LimitedTransferMiniMeToken(address _tokenFactory, address _parentToken, uint _parentSnapShotBlock, string _tokenName, uint8 _decimalUnits, string _tokenSymbol, bool _transfersEnabled) MiniMeToken(_tokenFactory, _parentToken, _parentSnapShotBlock, _tokenName, _decimalUnits, _tokenSymbol, _transfersEnabled) {
+    //constructor
+  }
+
+  function addLimitedTransferAddress(address _allowed) public onlyController returns (bool success) {
+    limitedTransferAddresses[_allowed] = true;
+    return true;
+  }
+
+  function setContractAddress(address _contractAddress) public onlyController returns (bool success) {
+    contractAddress = _contractAddress;
+    return true;
+  }
+
+  function enableLimitedTransfers(bool _limitedTransfersEnabled) public onlyController {
+    limitedTransfersEnabled = _limitedTransfersEnabled;
+    if (_limitedTransfersEnabled) {
+      super.enableTransfers(_limitedTransfersEnabled);
+    }
+  }
+
+  function approve(address _spender, uint256 _amount) public canTransfer(msg.sender, _spender) returns (bool success) {
```

Comment:
do we need to include the "canTransfer" as modifier? 
What if we always allow approve? 
Is there a use case where we need to transfer from an arbitrary account to the contract where the initiator (msg.sender) is not the contract?

The current flow only uses approve (from any account) and transferFrom/transfer (from contract)

Reply:
As long the transferFrom still has this modifier it should be fine indeed